SPEC.md

```diff
@@ -1041,7 +1061,7 @@ Example input:
 
 ```json
 {
-  "sum.ints": ["0", "1", "2"]
```

Comment:
According to [the section on task outputs](https://github.com/openwdl/wdl/blob/wdl-1.2/SPEC.md#task-outputs), this should be prefixed with the task name, right?